solr/CHANGES.txt

```diff
@@ -132,6 +132,8 @@ Optimizations
 
 * SOLR-17099: Do not return spurious tags when fetching metrics from a Solr node to another. (Pierre Salagnac)
 
+* SOLR-17269: Simplify cores of Coordinator node to avoid unnecessary registrations on Zookeeper (Patson Luk)
```

Comment:
Maybe:  Prevent the "Coordinator node" feature from registering synthetic cores in ZooKeeper.

I worded it this way as a non-user of this trying to understand it.